### PR TITLE
partial revert to fix compatible issue

### PR DESCRIPTION
**Pull request by wkpark**
* fix `submit()`, `submit_*()`, `create_submit_args()`
* ~~add `console.warn = function(){};` to ignore bunch of warnings~~ extracted to another PR.

these `submit*()` are used several extensions by means of hook or direct call.
this fix revert some part of gradio 4 fix to make it more compatible with A1111.
for example, uddetailer use `submit.apply()` call directly.

### CODE REVIEW

File: javascript/gradio.js [hunk 1]
```diff
@@ -5,3 +5,6 @@ Object.defineProperty(Array.prototype, 'toLowerCase', {
         return this;
     }
 });
+
+// disable console.warn
+console.warn = function(){};
```
Comment (chocolateboy) [inline javascript/gradio.js:10]: It would be better to fix the warnings rather than hiding them.

**Review by chocolateboy — CHANGES_REQUESTED** [review]

File: javascript/gradio.js (hunk 1 above)
Comment (wkpark) [reply javascript/gradio.js]: simply removed. `console.warn` is not related with this issue.

**Comment by lllyasviel** [comment]
hi do you have some code links to what might be using this to help understanding

**Comment by DenOfEquity** [comment]
Seems OK with txt2img and img2img, but broken with Extras - because Extras has three outputs instead of four.
Rather than having two versions of `create_submit_args`, or further modifying it to handle this case, IMO it would now be better to inline the fixes into `submit`, `submit_img2img`, and `submit_extras`; leaving the original function as is.

**Comment by wkpark** [comment]
as you know, originally, `create_submit_args()` simply nullify gallery output array to fix strage gradio behavior.
it works ok with the A1111 but with the webui-forge, the output garbage array make some trouble.
so I simply drop the output array as forge originally intended and fix compatible issue for some extensions.

**Pull request merged**